wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Sep12.kt

```diff
@@ -69,19 +73,23 @@ internal constructor(
    * @param memo (optional) the client-generated memo of type ID that uniquely identifies the
    * customer. If a memo is present in the decoded SEP-10 JWT's sub value, it must match this
    * parameter value.
+   * @param transactionId (optional) the ID of the transaction that the customer is being KYC'ed
+   * for.
    * @return a customer with id information
    */
   suspend fun add(
     sep9Info: Map<String, String>,
     memo: ULong? = null,
     type: String? = null,
+    transactionId: String? = null,
   ): AddCustomerResponse {
     val customer: MutableMap<String, String> = mutableMapOf()
 
     populateMap(type, customer, memo, sep9Info)
 
     val urlBuilder = URLBuilder(baseUrl)
     urlBuilder.appendPathSegments("customer")
+    urlBuilder.addParameter("transaction_id", transactionId)
```

Comment:
@philipliu @Ifropc shouldn't the `transaction_id` value go in the request `body` (aka _populateMap_) instead of a url parameter for this `PUT` request?

Reply:
Thanks for catching this. I'll publish a PR to fix this.